src/containers/Review/overview/Overview.tsx

```diff
@@ -1,26 +1,45 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { DateTime } from 'luxon'
-import { Header, Icon, Message, Segment } from 'semantic-ui-react'
+import { Header, Icon, Message, Segment, Button, Form } from 'semantic-ui-react'
 import { useLanguageProvider } from '../../../contexts/Localisation'
 import useLocalisedEnums from '../../../utils/hooks/useLocalisedEnums'
+import useConfirmModal from '../../../utils/hooks/useConfirmModal'
+import { postRequest } from '../../../utils/helpers/fetchMethods'
 import { FullStructure } from '../../../utils/types'
-import { ActivityLog, ApplicationOutcome } from '../../../utils/generated/graphql'
+import {
+  ActivityLog,
+  ApplicationOutcome,
+  ApplicationStatus,
+} from '../../../utils/generated/graphql'
+import config from '../../../config'
 
 export const Overview: React.FC<{
   structure: FullStructure
   activityLog: ActivityLog[]
 }> = ({
   structure: {
-    info: { current, outcome, user, org, serial, template },
+    info: { current, outcome, user, org, serial, id, template },
+    scheduledEvents,
+    reload,
   },
   activityLog,
 }) => {
   const { strings } = useLanguageProvider()
   const { Outcome } = useLocalisedEnums()
+  const [deadlineDays, setDeadlineDays] = useState(5)
+  const { ConfirmModal, showModal } = useConfirmModal()
   const applicant = `${user?.firstName} ${user?.lastName}`
   const organisation = org?.name
   const { started, completed } = getDates(activityLog)
   const stage = current.stage.name
+  const applicantDeadlineEvent = scheduledEvents.filter(
```

Comment:
I feel like this should be extracted in structure, i.e. structure should have `applicationDeadlineEvent:? {isActive, deadline}`

Reply:
Okay, I think this works (for both this and the previous comment): 95a4f2cd